client/packages/common/src/hooks/useUrlQuery/useUrlQuery.ts

```diff
@@ -64,7 +65,12 @@ export const useUrlQuery = ({ skipParse = [] }: useUrlQueryProps = {}) => {
 
     Object.entries(values).forEach(([key, value]) => {
       if (!value) delete newQueryObject[key];
-      else {
+      else if (key === 'tab') {
```

Comment:
Common filter params stay in the url when changing tabs and makes the search all funky

Reply:
Think this shouldn't go in this `common` hook. Since this is very application specific. Is there a way to handle this at a higher level? One option was to remember the tab filters in the local storage:
https://github.com/openmsupply/open-msupply/pull/2237#discussion_r1328173383

However, maybe in the linked PR has a solution to clear the filter?